.github/PULL_REQUEST_TEMPLATE/lightweight.md

```diff
@@ -19,6 +19,7 @@ _Summary of the purpose of this pull request._
 _These tasks are for the pull request creator to tick off._
 
 - [ ] Developer testing
+- [ ] Make sure relevant changelog is updated if any changes affect users
```

Comment:
Can you link to the changelogs in the PR description to make it more convenient? And can you move Developer Testing to the Reviewer Tasks here? (in all PR templates)

Reply:
Added. Can't test PR template relative links are correct without merge 😞